src/borg/testsuite/hashindex.py

```diff
@@ -5,6 +5,8 @@
 import tempfile
 import zlib
 
+import pytest
```

Comment:
# Note: these tests are part of the self test, do not use or import py.test functionality here.
 #       See borg.selftest for details. If you add/remove test methods, update SELFTEST_COUNT